File: tuts/011-getting-started-batch-fargate/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from batch_wrapper import BatchWrapper


def run_scenario():
    client = boto3.client("batch")
    wrapper = BatchWrapper(client)
    suffix = uuid.uuid4().hex[:8]
    print(f"Running Batch getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()

File: tuts/011-getting-started-batch-fargate/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 'batch_wrapper'

def run_scenario
  client = Aws::Batch::Client.new
  wrapper = BatchWrapper.new(client)
  puts "Running Batch getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME
